app/src/main/java/org/oppia/android/app/settings/profile/DeleteProfileSuccessDialogFragment.kt

```diff
@@ -0,0 +1,44 @@
+package org.oppia.android.app.settings.profile
+
+import android.app.Dialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import org.oppia.android.R
+import org.oppia.android.app.administratorcontrols.AdministratorControlsActivity
+import org.oppia.android.app.fragment.FragmentComponentImpl
+import org.oppia.android.app.fragment.InjectableDialogFragment
+
+class DeleteProfileSuccessDialogFragment : InjectableDialogFragment() {
+
+  companion object {
+    const val TAG = "TAG_DELETE_DIALOG_FRAGMENT"
```

Comment:
Please have a look at other similar fragments/dialogfragment in code to see how we name our TAGs

https://github.com/oppia/oppia-android/blob/5801df02317cc3f052d79a3dea8b3becd06bc60d/app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragment.kt#L25

Reply:
Updated this

Reply:
This change is still pending

Reply:
Updated

Reply:
It should be 
`const val DELETE_PROFILE_SUCCESS_DIALOG_FRAGMENT_TAG = "DELETE_PROFILE_SUCCESS_DIALOG_FRAGMENT"`

Reply:
Updated this